### ISSUE TITLE
Maya: Deadline asset dependencies not working properly

### ISSUE THREAD

**Issue by m-u-r-p-h-y** [ynput/OpenPype#3332]
**Describe the bug**

We have a feature of checking asset dependencies turned on by default for Maya jobs submitted to Deadline

![image](https://user-images.githubusercontent.com/352795/173393994-d32c6360-69f9-4c6d-99f9-1e0ec316cbfc.png)
 

In many cases, it is creating issues with checking all the assets used in the Maya scene and not starting the render indefinitely waiting for dependencies to be fulfilled.

Maybe the issue is with the tokens in file names
![image](https://user-images.githubusercontent.com/352795/173396801-10de3805-a3fb-4239-a88e-3ae483c58d3c.png)

I would definitely suggest turning this off by default and leaving it optional as it is creating more troubles at the moment and it makes sense only in cases when uploading the render scene to the cloud or other RARE cases.

[cuID:[OP-3413](https://app.clickup.com/t/2fexvep)]

**Comment by BigRoy** [comment]
I agree that it's given us more trouble than it's worth too. :)

**Comment by BigRoy** [comment]
Also we have had cases with an empty entry popping up in the list with both asset name and full path just being empty. Not sure why yet - but those also keep the job from starting.

**Comment by BigRoy** [comment]
Was this issue resolved with this PR https://github.com/ynput/OpenPype/pull/4242?

**Comment by m-u-r-p-h-y** [comment]
> Was this issue resolved with this PR #4242?

I don't think so. There was a different issue of setting wrong dependency between render and tile assembly job. (assembly job should be dependent on render job to finish first)
